python/ray/node.py

```diff
@@ -944,15 +1018,43 @@ def start_ray_client_server(self):
             process_info
         ]
 
+    def _write_cluster_info_to_kv(self):
+        client = self.get_gcs_client()
+
+        # Job counter.
+        # TODO(mwtian): Fix job counter usage in C++ to support GCS KV.
+        client.internal_kv_put(
+            b"JobCounter",
+            int(0).to_bytes(8, "little"),
+            overwrite=True,
+            namespace=ray_constants.KV_NAMESPACE_CLUSTER)
```

Comment:
Question: if it's redis based, I don't think there is namespace in the key (so cpp code need to be updated).
If it's memory-based, I feel we don't need this, since it's already handled right?

Reply:
Good point. I double checked: Redis store uses `INCR JobCounter` command which automatically initializes `JobCounter` to 0; inmemory store uses a memory variable for `JobCounter`. I removed this section.